package.json

```diff
@@ -5,7 +5,6 @@
   "main": "index.js",
   "types": "index.d.ts",
   "scripts": {
-    "prepare": "rm -rf example",
```

Comment:
I believe it was needed for the `example` not to be included in the release? Or am I missing something?

Reply:
Probably but it runs on every `yarn install` so the `Example` directory will be always removed